websites/I/IDLIX/metadata.json

```diff
@@ -8,20 +8,16 @@
 	"description": {
 		"en": "Watch the latest movies, TV series, dramas, anime with high quality available in Indonesian subtitles and updated every day. Box Office Movies to the Best TV Series are all available on the IDLIX site."
 	},
-	"url": "tv.idlixofficial.co",
-	"version": "1.0.9",
+	"url": "idlix.online",
+	"regExp": "^([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(?<!172.(16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31))(?<!127)(?<!^10)(?<!^0).([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(?<!192.168)(?<!172.(16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31)).([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5]).([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$|((tv([0-9])[.])?id(f)?lix(official)?[.][a-z]{2,6})",
```

Comment:
Use [htis regex](https://stackoverflow.com/questions/33453057/regex-to-only-match-public-ipv4-address) to match all public ips (bcuz they use random public ips.)